test/initializeTests.js

```diff
@@ -45,7 +45,9 @@ export async function initializeTestService(useLive = false, isAdmin = false) {
     isMA: true,
   }
   initializeService(config)
-  let mock = jest.spyOn(railContentModule, 'fetchUserPermissionsData')
-  let testData = { permissions: [78, 91, 92], isAdmin: isAdmin }
-  mock.mockImplementation(() => testData)
+
+  // Mock user permissions
+  let permissionsMock = jest.spyOn(railContentModule, 'fetchUserPermissionsData')
+  let permissionsData = { permissions: [78, 91, 92], isAdmin: isAdmin }
```

Comment:
Only permissions left after migration will be 
base: 91
plus: 92
free: 134
lifetime: 108
```suggestion
  let permissionsData = { permissions: [108, 91, 92], isAdmin: isAdmin }
```